platform/libs.javafx/manifest.mf

```diff
@@ -38,5 +38,4 @@ OpenIDE-Module-Provides: javafx.animation,
     javafx.util,
     javafx.util.converter,
     netscape.javascript
-Class-Path: ${java.home}/lib/ext/jfxrt.jar
-
+Class-Path: %24%7Bjava.home%7D%2F/lib/ext/jfxrt.jar
```

Comment:
Well, this is ugly. Cant we just have JAVA_HOME/lib/ext/jfxrt.jar ?

Reply:
Very ugly, I agree. I was also considering defining some new token instead of `${java.home}`, but that would mean API changes, versioning, etc. At the end I decided to apply a [simple bugfix](https://github.com/apache/netbeans/pull/2761#discussion_r577187116).